src/classes/HH_Container_LCTRL.cls

```diff
@@ -327,7 +329,6 @@ public with sharing class HH_Container_LCTRL {
     * @description deletes the Contacts in the database
     * @param listCon The list of Contacts to delete
     * @return void
-    */
```

Comment:
I wanted to keep it around in case PM decides to offer both types of functionality (delete and remove).  Mary and I had talked about maybe adding a backlog item for that (though I don't recall what we decided).